<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="videoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="ScaledView" name="glWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCam1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cam1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCam2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cam2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCam3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cam3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCam4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cam4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAllCam">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>AllCam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZoom2x">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2x Zoom</string>
    </property>
   </widget>
   <widget class="QLabel" name="smallVideoLabel1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>390</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="smallVideoLabel2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>390</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="smallVideoLabel3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>390</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="smallVideoLabel4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>390</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>videoLabel</zorder>
   <zorder>btnCam1</zorder>
   <zorder>btnCam2</zorder>
   <zorder>btnCam3</zorder>
   <zorder>btnCam4</zorder>
   <zorder>btnAllCam</zorder>
   <zorder>btnZoom2x</zorder>
   <zorder>smallVideoLabel1</zorder>
   <zorder>smallVideoLabel2</zorder>
   <zorder>smallVideoLabel3</zorder>
   <zorder>smallVideoLabel4</zorder>
   <zorder>glWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScaledView</class>
   <extends>QWidget</extends>
   <header>scaledview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
